webpack.config.js

```diff
@@ -1,10 +1,17 @@
-const VueLoaderPlugin = require('vue-loader/lib/plugin'),
-  env = process.env.NODE_ENV || 'development';
+const path = require('path'),
+  VueLoaderPlugin = require('vue-loader/lib/plugin'),
+  libraryName = 'cache-clear',
+  outputFilename = process.env.NODE_ENV === 'production'
+    ? `${libraryName}.min.js`
```

Comment:
but the version will be minimized without the plugin?

Reply:
Yes! Webpack has it's own minification strategy when running in production mode, please see here: https://webpack.js.org/guides/production/#minification

Reply:
Thanks for sharing 👍